core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1672,19 +1674,17 @@ public function deleteChilds($table, $id, &$delete)
 	 */
 	public function undo()
```

Comment:
shouldn't we check permissions here if the user is allowed to read the `tl_undo` record and write the restored record?